src/create_disk.sh

```diff
@@ -476,23 +480,10 @@ generate_gpgkeys() {
 # shellcheck disable=SC2031
 case "$arch" in
 x86_64)
-    # UEFI
-    install_uefi
-    if [ "${x86_bios_bootloader}" = 1 ]; then
-        # And BIOS grub in addition.  See also
-        # https://github.com/coreos/fedora-coreos-tracker/issues/32
-        # Install BIOS/PReP bootloader using the target system's grub2-install,
-        # see https://github.com/coreos/coreos-assembler/issues/3156
-        chroot_run /sbin/grub2-install \
-            --target i386-pc \
-            --boot-directory $rootfs/boot \
-            --modules mdraid1x \
-            "$disk"
-    fi
+    install_via_bootupd
     ;;
 aarch64)
-    # Our aarch64 is UEFI only.
-    install_uefi
+    install_via_bootupd
     ;;
 ppc64le)
```

Comment:
without ppc64le support in bootupd I think it might make our "implement this in OSBuild" story a bit more complicated. Are we considering adding ppc64le support to bootupd?

Reply:
Ah right, sorry I forgot we actually *did* add PReP support in that same PR, so I've amended the PR (untested) to also add that.

And now that we've done that, maybe we should go ahead and add the (really trivial) support for zipl...

Reply:
Oh right, zipl isn't trivial at all...lots of logic in https://github.com/coreos/coreos-installer/blob/main/src/s390x/zipl.rs

Which gets into a tricky problem in having bootupd depend on coreos-installer (or worse, bootc depend on it).

We probably need to factor that logic out into a crate.

Reply:
The more general problem is that s390x/zipl actually mainly ties together the kernel/initramfs/bootloader - which in many ways is a good thing - it's actually what we *also* want to support for a UKI-style flow, but *that* then merges the problem domain of bootloader and OS update.

Reply:
> Ah right, sorry I forgot we actually _did_ add PReP support in that same PR, so I've amended the PR (untested) to also add that.

Oh nice. I didn't catch that 👍